### PR TITLE
Add scope filter to lerna bootstrap command

### LINKED ISSUE #4739: npm install fails
### Core Library

MSAL.js v2 (@azure/msal-browser)

### Core Library Version

n/a

### Wrapper Library

Not Applicable

### Wrapper Library Version

none

### Description

Following this document to do PR to contribute.
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/contributing.md

and "Npm install" fails with following error.
image

and npm build fails as well.
image

Jason Nutter mentioned, validation library needs to be removed.
image

### Error Message

this is with NPM install, so N/A

### Msal Logs

this is with NPM install, so N/A

### MSAL Configuration

```javascript
this is with NPM install, so N/A
```

### Relevant Code Snippets

```javascript
this is with NPM install, so N/A
```

### Reproduction Steps

Fork the "microsoft-authentication-library-for-js.git" and do "npm install"

### Expected Behavior

Npm install and npm build success, and able to run npm test.

### Identity Provider

Other

### Browsers Affected (Select all that apply)

Other

### Regression

_No response_

### Source

Internal (Microsoft)

### PR DESCRIPTION
**Pull request by hectormmg**
- Adds scope filter to bootstrap command in lerna.json to remove unpublished libraries
- Fixes #4739

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -22,6 +22,17 @@
...
         "stream": true
+    },
+    "bootstrap": {
+        "scope": [
+            "msal",
+            "@azure/msal-browser",
+            "@azure/msal-common",
+            "@azure/msal-node",
+            "@azure/msal-react",
+            "@azure/msal-angular"
```
Comment (tnorling) [inline lerna.json:33]: msal-node-extensions?

Comment (bmahall) [reply lerna.json:33]: once token-validation is published, does the user need to do extra steps to build?

Comment (jasonnutter) [reply lerna.json:33]: node-extensions isnt included in any of the other commands currently.

Comment (jasonnutter) [reply lerna.json:33]: > once token-validation is published, does the user need to do extra steps to build?

Once it is published, we can add it into this list.

**Review by jasonnutter — APPROVED** [review]

File: lerna.json (hunk 1 above)
Comment (tnorling) [reply lerna.json:33]: > node-extensions isnt included in any of the other commands currently.

The other commands are arguably not as important as the bootstrap. node-extensions should still consume the local copy of msal-common

Comment (hectormmg) [reply lerna.json:33]: @tnorling adding it, thanks!

**Review by bmahall — APPROVED** [review]
Looks good, had some questions on the side , can discuss it offline!

**Review by peterzenz — APPROVED** [review]

**Pull request merged**